src/components/features/Order/MessageSection/index.tsx

```diff
@@ -1,32 +1,36 @@
-import { Box, Flex, forwardRef, Text, Textarea } from '@chakra-ui/react';
+import { Box, Flex, Text, Textarea } from '@chakra-ui/react';
+import { useFormContext } from 'react-hook-form';
 
-export const MessageSection = forwardRef((_, ref) => (
-  <Box as="section" w="100%" padding="44px 0px 32px">
-    <Flex w="100%" justifyContent="center">
-      <Text
-        fontSize="18px"
-        lineHeight="21px"
-        color="rgb(34, 34, 34)"
-        fontWeight={700}
-        boxSizing="border-box"
-      >
-        나에게 주는 선물
-      </Text>
-    </Flex>
-    <Box w="100%" padding="14px 30px">
-      <Box w="100%" padding="12px 30px 16px">
-        <Textarea
-          ref={ref}
-          name="messageCardTextMessage"
-          placeholder="선물과 함께 보낼 메시지를 적어보세요"
-          w="100%"
-          h="100px"
-          fontSize="1rem"
-          borderRadius="0.375rem"
-          outlineOffset="2px"
-          background="#EDF2F7"
-        />
+export const MessageSection = () => {
+  const { register } = useFormContext();
+
+  return (
+    <Box as="section" w="100%" padding="44px 0px 32px">
+      <Flex w="100%" justifyContent="center">
+        <Text
+          fontSize="18px"
+          lineHeight="21px"
+          color="rgb(34, 34, 34)"
+          fontWeight={700}
+          boxSizing="border-box"
+        >
+          나에게 주는 선물
+        </Text>
+      </Flex>
+      <Box w="100%" padding="14px 30px">
+        <Box w="100%" padding="12px 30px 16px">
+          <Textarea
+            {...register("cardMessage")}
```

Comment:
```suggestion
            {...register("cardMessage", { required: true })}
```

필수값이라면 react-hook-form에서 validation을 하도록 옵션을 줄 수 있습니다. 이를 통해 submit handler에서 따로 검사하지 않고 `formState.errors`를 통해 에러 여부를 참조해서 alert가 아니라 jsx를 통해 뷰로 표현해줄 수 있으니까요.